src/zero/api/spec.clj

```diff
@@ -1,103 +1,107 @@
 (ns zero.api.spec
   "Define specs used in routes"
-  (:require [zero.service.cromwell              :as cromwell]
-            [clojure.spec.alpha                 :as s]
-            [zero.util                          :as util])
+  (:require [zero.service.cromwell :as cromwell]
+            [clojure.spec.alpha :as s]
+            [zero.util :as util])
   (:import [java.util UUID]))
 
 (defn uuid-string? [s] (uuid? (util/do-or-nil (UUID/fromString s))))
 
-(s/def ::base_file_name       string?)
-(s/def ::commit               (s/and string? (comp (partial == 40) count)))
-(s/def ::created              inst?)
-(s/def ::creator              string?)
-(s/def ::cromwell             string?)
-(s/def ::end                  string?)
-(s/def ::environment          string?)
+;; shared
+(s/def ::environment string?)
```

Comment:
I like how you grouped the specs! I think alphabetizing them would be nice as well!

Reply:
Nice suggestion! Let me alphabetize them within the current groups.